Training/1.0 Introduction.md

```diff
@@ -11,38 +11,83 @@ First we we learn a little bit about coding in Python, although many of you will
 
 List all the Data Types of Python with examples.  
 
-```
-
-```
+``` str = 'word'
+``` int = 2
+``` float = 2.1
+``` complex = 2j
+``` list = ["hello", "goodbye"]
+``` tuple = ("hello", "goodbye")
+``` range = range(2)
+``` dict = {"eat":"chips", "drink":"water"}
+``` set = {"hello", "goodbye"}
+``` frozenset = ({"hello", "goodbye"})
+``` bool = True
+``` bytes = b"Cheers"
+``` bytearray = bytearray(2)
+``` memoryview = memoryview(bytes(2))
+``` NoneType = None
 
 ### Task 2: 
 
 Other useful ways of storing data are in List, Arrays and Tuples. Create one of each and assign them to the variables a, b and c respectively. 
 
-```
-
+``` listx = ["a", "b", "c"]
+``` arrayx = listx
+``` tuplex =("a", "b", "c")
 ```
 
 ### Task 3: 
 
 We are also interested in becoming good software developers so we will need to use conditional, loops. Write an statement where the summation of a variable x+1 will be calculated if the value of x is greater than 2. 
 
-```
+``` x = input("please input a number")
+``` x = float(x)
+``` if 2<<(x+1):
+```     y = x+1
+``` else:
+        continue
+    print(y)
```

Comment:
Good attempt but a couple of things to keep in mind:
1) Task says calculate summation if x is greater than 2, not if x+1 is greater than 2.
2) If you check the python documentation, operator `<<` is not the same as `<`, so be careful with how you use them.